### ISSUE TITLE
The performances of mask rcnn 

### ISSUE THREAD

**Issue by YutingXiao** [qqlu/Amodal-Instance-Segmentation-through-KINS-Dataset#14]
I saw that the mAP performance of mask rcnn is 29.3(amodal seg) and 26.6(inmodal seg) in cvpr2019 paper. But there are many mask rcnn config files in the code, So which version of mask rcnn corresponds to this 29.3/26.6 performance???

**Comment by qqlu** [comment]
You could use Amodal-Instance-Segmentation-through-KINS-Dataset/Reference Code/configs/kitti/amodalinmodalseg_split_amodal_class_weight6_feature_fusion.yaml

**Comment by qqlu** [comment]
And this is the same hyper-parameter setting for mask R-CNN.

**Comment by YutingXiao** [comment]
Have you used adam optimizer on mask rcnn, I saw that there is adam option in your code, how is the performance when optimized by adam?

**Comment by qqlu** [comment]
Hi, I did not use the adam optimizer in this paper. And I cannot remember the performance of the KINS when used the adam optimizer. For the traditional COCO dataset, there is a little performance improvement like 0.3-0.4 for instance segmentation and no improvement for object detection. You could check it by default Mask R-CNN in Detectron2. The setting is that changing the learning rate of Adam to 0.00002 compared with 0.02 in SGD.
